changelog/Microsoft.DirectoryServices.json

```
@@ -18,6 +18,48 @@
       "WorkloadArea": "Applications",
       "SubArea": ""
     },
+    {
+      "ChangeList": [
+        { 
+          "Id": "12f096a3-6bd8-49db-8753-24c9272a288b",
+          "ApiChange": "Property",
+          "ChangedApiName": "IsMemberManagementRestricted",
+          "ChangeType": "Addition",
+          "Description": "Added the **IsMemberManagementRestricted** property to the [administrativeUnit](https://docs.microsoft.com/en-us/graph/api/resources/administrativeunit?view=graph-rest-beta) resource.",
```

Suggestion to enhance the description. Please tweak where applicable.
```suggestion
          "Description": "Added the **isMemberManagementRestricted** property to the [administrativeUnit](https://docs.microsoft.com/en-us/graph/api/resources/administrativeunit?view=graph-rest-beta) resource. Use this property to define administrative units whose management is restricted to a limited group of administrators with the appropriate permissions and specific Azure AD administrative roles.",
```

---

....to define admins units whose roles don't inherit from tenant level administrators, and management of individual member objects is limited to admins scoped over a restricted management admin unit.

---

I dont think we should add specific info in the changelog that could out of date. Please see updated description in administrativeunit.md